pymbar/mbar.py

```diff
@@ -278,25 +278,17 @@ def __init__(self, u_kn, N_k, maximum_iterations=10000, relative_tolerance=1.0e-
                 print("f_k = ")
                 print(self.f_k)
 
-        for solver in subsampling_protocol:
-            if 'options' not in solver:
-                solver['options'] = dict()  # make sure there is SOME dictionary in for options.
-                if self.verbose:
-                    # should add in other ways to get information out of the scipy solvers, not just adaptive,
-                    # which might involve passing in different combinations of options, and passing out other strings.
-                    if solver['method'] == 'adaptive':
-                        solver['options']['verbose']=True
-
+        if solver_protocol is None:
+            solver_protocol = ({'method': None},)
         for solver in solver_protocol:
             if 'options' not in solver:
                 solver['options'] = dict()
-                if self.verbose:
-                    # should add in other ways to get information out of the scipy solvers, not just adaptive,
-                    # which might involve passing in different combinations of options, and passing out other strings.
-                    if solver['method'] == 'adaptive':
```

Comment:
How is the default set here?

Reply:
No, I suppress the warnings thrown by not having the `verbose` options in the SciPy solvers, but explicitly only that.

Reply:
Ah, I see.  Missed that logic. SEEMS like it's better to handle it explicitly than potentially mask other warning problems?